tensorflow_datasets/testing/fake_data_utils.py

```diff
@@ -66,3 +69,33 @@ def get_random_png(height=None, width=None, channels=CHANNELS_NB):
   fobj.write(res)
   fobj.close()
   return fobj.name
+
+
+def get_random_audio(sample = SAMPLE_RATE):
+  """Returns random audio as np.ndarray (float32)."""
+  sample_number = np.arange(AUDIO_DURATION * sample)
+  waveform = np.sin(2 * np.pi * each_sample_number * 440.0 / sample).astype(np.float32)
+  waveform = waveform * 0.3
+  return waveform
+
+def get_random_wav_C1(channels=1, sample = SAMPLE_RATE):
+  """Returns path to WAV audio having channels = 1."""
+  audio = get_random_audio(sample).reshape(-1,channels)
+  wav = tf.audio.encode_wav(audio, sample)
+  with utils.nogpu_session() as sess:
+    res = sess.run(wav)
+  fobj = tempfile.NamedTemporaryFile(delete=False, mode='wb', suffix='.wav')
+  fobj.write(res)
+  fobj.close()
+  return fobj.name
+
+def get_random_wav_C2(channels=2, sample = SAMPLE_RATE):
+  """Returns path to WAV audio having channels = 2."""
+  audio = get_random_audio(sample).reshape(-1,channels)
+  wav = tf.audio.encode_wav(audio, sample)
+  with utils.nogpu_session() as sess:
+    res = sess.run(wav)
+  fobj = tempfile.NamedTemporaryFile(delete=False, mode='wb', suffix='.wav')
```

Comment:
It is best to use context manager instead as it automatically close the file and do the cleanup:

```
with tempfile.NamedTemporaryFile() as f:
  ...
```

Reply:
So should I change it or remains the same.